### DISCUSSION TITLE
Save media files

### DISCUSSION THREAD

**Discussion by roabiss** [vasani-arpit/WBOT#342] [Q&A]
Hello, is it possible to save media files in folders by contact name?

**Comment by vasani-arpit** [comment]
Yes. With a small code change.

**Reply by roabiss** [reply]
Can you help me with this? 
I receive a lot of documents and arranged in folders would be a real help for me!

**Reply by vasani-arpit** [reply]
1. Run the wbot locally 
2. see this line. https://github.com/vasani-arpit/WBOT/blob/master/src/index.js#L149
3. it is responsible for file path just replace the first ```+``` with ```,``` 

That's it.
